fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java

```diff
@@ -104,7 +104,10 @@ public static void estimate(GroupExpression groupExpression) {
 
     private void estimate() {
         StatsDeriveResult stats = groupExpression.getPlan().accept(this, null);
-        groupExpression.getOwnerGroup().setStatistics(stats);
+        if (groupExpression.getOwnerGroup().getStatistics() == null
+                || (stats.getRowCount() < groupExpression.getOwnerGroup().getStatistics().getRowCount())) {
```

Comment:
add comment to explain `stats.getRowCount() < groupExpression.getOwnerGroup().getStatistics().getRowCount())`

Reply:
done